openpower/scripts/firmware-whitelist

```diff
@@ -10,6 +10,7 @@ whitelist=(     'acenic/'
                 'cxgb4/t4fw-1.16.63.0.bin'
                 'cxgb4/t4fw.bin'
                 'cxgb3/'
+                'qed/'
```

Comment:
Instead of including all of the qed directory, specify the file that is required by the current kernel (4.19). This looks to be 8.37.2.0, so we want `qed/qed_init_values_zipped-8.37.2.0.bin`.

Reply:
Yes, it is. But the problem is in buildroot, it still use 8.37.2.0:

buildroot/package/linux-firmware/linux-firmware.mk:
```
ifeq ($(BR2_PACKAGE_LINUX_FIRMWARE_QLOGIC_4X),y)
LINUX_FIRMWARE_FILES += \
	qed/qed_init_values_zipped-8.33.11.0.bin
endif
```

Reply:
Of course, I can set LINUX_FIRMWARE_FILES through environment variables, but it doesn't resolve the problem when buildroot contains old linux-firmware package without firmware defined for current kernel. Force set of LINUX_FIRMWARE_VERSION also cannot be used because I have to modify the hash file of linux-firmware that belongs to buildroot.

Reply:
https://github.com/open-power/op-build/pull/2524